### PR TITLE
Add `SHOW METRICS INFO` query

### PR DESCRIPTION
**Pull request by Josipmrden**
### Description

New query has been added `SHOW METRICS INFO`, which expose the same metrics like from the HTTP endpoint, over a query

### Pull request type

Please check what kind of PR this is:

- [X] Fix or improvement of an existing page
- [ ] New documentation page, release related

### Related PRs and issues

PR this doc page is related to: 
https://github.com/memgraph/memgraph/pull/2052

Closes:
X

### Checklist:

- [X] Check all content with Grammarly
- [X] Perform a self-review of my code
- [X] Make corresponding changes to the rest of the documentation (consult with the DX team)
- [X] The build passes locally
- [X] My changes generate no new warnings or errors
- [X] Add a corresponding label
- [X] If release-related, add a product and version label
- [X] If release-related, add release note on product PR

### CODE REVIEW

**Review by hal-eisen-MG — APPROVED** [review]
LGTM

**Review by kgolubic — APPROVED** [review]
Everything looks OK to me. Can be merged right after the memgraph development PR is merged.

**Comment by hal-eisen-MG** [comment]
Please link the blocking PR

**Pull request merged**
